ionos_dev/analytics/superset/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: superset-ingress
+  namespace: analytics
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-dev-issuer
+    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: analytics.dome-marketplace-dev.org
```

Comment:
Should it be analytics.dome-marketplace-dev2.org

Reply:
Thats a deleted branch/PR. 
The valid PR is already merged with main. There are no analytics folder on "ionos_dev".
I'll check if there is a related/same problem on main branch on dev2 analytics folder and I'll create a new branch/PR in that case. Thanks.